src/main/kotlin/org/wa/rceditor/application/view/fragments/NewProjectDialog.kt

```diff
@@ -0,0 +1,113 @@
+package org.wa.rceditor.application.view.fragments
+
+import com.jfoenix.controls.JFXDialog
+import com.jfoenix.controls.JFXListView
+import com.jfoenix.controls.JFXTextField
+import javafx.beans.property.SimpleObjectProperty
+import javafx.collections.ObservableList
+import javafx.scene.layout.VBox
+import org.wa.rceditor.application.Styles
+import org.wa.rceditor.application.model.ProjectItem
+import org.wa.rceditor.application.model.ProjectItemModel
+import org.wa.rceditor.application.model.SourceItem
+import org.wa.rceditor.application.model.SourceItemModel
+import tornadofx.*
+
+class NewProjectDialog(listView: JFXListView<ProjectItem>, dialog: JFXDialog): VBox() {
+    var listView: JFXListView<ProjectItem> by singleAssign()
+    var dialog: JFXDialog by singleAssign()
```

Comment:
Is this dialog being shown anywhere? I only saw that it was being closed in the commitAll function

Reply:
@mbr4477 it's being shown in here https://github.com/mXaln/RCEditor/blob/material_design/src/main/kotlin/org/wa/rceditor/application/view/MainView.kt#L362